pyproject.toml

```diff
@@ -37,24 +145,18 @@ known-first-party = ["basket"]
 [tool.tox]
 legacy_tox_ini = """
 [tox]
-envlist = py37,py38,py39,py310,py37-lint
+envlist = py37,py38,py39,py310,py311
 
 [gh-actions]
 python =
     3.7: py37
     3.8: py38
     3.9: py39
     3.10: py310
+    3.11: py311
 
 [testenv]
 deps = -rrequirements/dev.txt
 commands =
-    pytest basket/tests.py
-
-[testenv:py37-lint]
-basepython = python3.7
-changedir = {toxinidir}
-commands = 
-    black --check basket setup.py
-    ruff basket setup.py
-"""
\ No newline at end of file
+    pytest tests/
```

Comment:
I would put the `tests/` thing in a `tool.pytest.ini_options` table in the file. Then here, I would have:

```
pytest {posargs}
```
```